lib/SILGen/SILGenFunction.cpp

```diff
@@ -895,3 +895,19 @@ SILGenBuilder::createAllocExistentialBox(SILLocation Loc,
                                                ConcreteType,
                                                Conformances);
 }
+
+void SILGenFunction::checkForImportedUsedConformances(Type type) {
+  // Recognize _BridgedNSError, which must pull in its witness table for
+  // dynamic casts to work
+  if (auto bridgedNSErrorProtocol =
+          getASTContext().getProtocol(KnownProtocolKind::BridgedNSError)) {
+    if (auto nominalDecl = type->getAnyNominal()) {
+      auto conformances = nominalDecl->getAllConformances();
```

Comment:
I was avoiding using that method because I didn't know which Module to supply (e.g. the one for Foundation, or the one where the enum was perhaps imported into?). The comments for the method do mention that it is ultimately ignored, but I didn't want sudden breakage as soon as the FIXME was resolved. Also, I understood the method's implementation far less. Which Module do you recommend I pass in?